src/utils/sentry.js

```diff
@@ -0,0 +1,19 @@
+import * as Sentry from '@sentry/browser';
+
+import config from 'config';
+import { PRODUCTION } from 'constants/env';
+
+export const initSentry = () => {
```

Comment:
Use the `function` keyword while exporting a utility function for consistency across the application and add a docblock.
Also, change the function name to `init` since `sentry` is already in the utility name.
And  use it as
```js
import * as sentry from 'utils/sentry';

sentry.init() 
```
```suggestion
/**
* Initialize sentry.
*/
export function init() {
```

Reply:
ok got it dai!